bindings/python/cntk/layers/sequence.py

```diff
@@ -299,8 +299,24 @@ def recurrence_from_2(h, c, x):
         return _recurrence_from_n(h, c, x)
     def recurrence_from_3(h, c, a, x):
         return _recurrence_from_n(h, c, a, x)
+    def recurrence_from_4(h, c, a, b, x):
+        return _recurrence_from_n(h, c, a, b, x)
+    def recurrence_from_5(h, c, a, b, d, x):
+        return _recurrence_from_n(h, c, a, b, d, x)
+    def recurrence_from_6(h, c, a, b, d, e, x):
+        return _recurrence_from_n(h, c, a, b, d, e, x)
+    def recurrence_from_7(h, c, a, b, d, e, f, x):
+        return _recurrence_from_n(h, c, a, b, d, e, f, x)
+    def recurrence_from_8(h, c, a, b, d, e, f, g, x):
+        return _recurrence_from_n(h, c, a, b, d, e, f, g, x)
+    def recurrence_from_9(h, c, a, b, d, e, f, g, i, x):
+        return _recurrence_from_n(h, c, a, b, d, e, f, g, i, x)
+    def recurrence_from_10(h, c, a, b, d, e, f, g, j, x):
```

Comment:
missing i?

Reply:
Hi, sorry for the gross error. Amended and this good to go!